tests/kuttl/tests/dataplane-deploy-global-service-test/00-assert.yaml

```diff
@@ -1,3 +1,14 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 600
+collectors:
+- type: command
+  command: oc get OpenStackDataPlaneDeployment -n openstack edpm-compute-global -o yaml
```

Comment:
on thing im not sure about is if in ci we are expecting to use oc or kubectl I'm also not sure if its available in the default path or if we need to pass the kubeconfig.

we wont really know any of that until we see this run in the ci